server/api/user/user.controller.js

```diff
@@ -340,3 +347,43 @@ exports.removeTech = function(req,res){
         }
     });
 };
+
+/**
+ * Set reset token for user and email it the password token will expire after 24 hours.
+ */
+exports.resetPassword = function(req, res){
+    var userEmail = req.body.email;
+    User.findOne({
+        email: userEmail.toLowerCase()
+    }, function (err, user){
+        if (err) return res.json(401, err);
+        if (!user) return res.json(401, {message: 'User does not exist'});
```

Comment:
There is no password entered here- the only way to invoke the error is to not have the email exist (so it doesn't really matter what the error message is- they have the code as this is an OS project)

I think the solution to this is to have a captcha, which can be a separate issue.